message-transport/pkg/transport/metrics/consumer.go

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2018-Present the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package metrics
+
+import (
+	"github.com/projectriff/riff/message-transport/pkg/message"
+	"github.com/projectriff/riff/message-transport/pkg/transport"
+	"encoding/json"
+	"log"
+	"time"
+)
+
+const consumerSource = "consumer"
+
+// ConsumerAggregateMetric represents the reception of a number of message from a topic by a function in a time interval.
```

Comment:
small typo: "number of message", should be "messages"

Reply:
Fixed. I like attention to detail.